config/initializers/split.rb

```diff
@@ -0,0 +1,3 @@
+Split::Dashboard.use Rack::Auth::Basic do |username, password|
+  username == 'test' && password == 'test'
```

Comment:
Can you set these to environment variables? Maybe something like split_username and split_password?